Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -129,5 +129,17 @@ private void UpdateStabilizationPlane()
                 HolographicSettings.SetFocusPointForFrame(newFocusPointPosition, -gazeDirection);
             }
         }
+
+        /// <summary>
+        /// Sends message to the object currently focused on.
+        /// </summary>
+        /// <param name="message">Message to send</param>
+        public void SendMessageToFocusedObject(string message)
```

Comment:
A thought - maybe not as bad of a leak, but doesn't SendMessageToFocusedObject leak the abstraction similarly? It requires that GazeManager know the focused object and that the object it tracks is capable of receiving Unity style messages. This feels like the same contract as exposing the data through a readonly property.